ansible_collections/arista/avd/docs/installation/collection-installation.md

```diff
@@ -77,45 +77,30 @@ collections_paths = ${PWD}/collections:~/.ansible/collections:/usr/share/ansible
 $ ansible-galaxy collection install -U arista.avd
 Process install dependency map
 Starting collection install process
-Installing 'arista.avd:4.4.0' to '/root/.ansible/collections/ansible_collections/arista/avd'
+Installing 'arista.avd:4.8.0' to '/home/arista/.ansible/collections/ansible_collections/arista/avd'
 ```
 
-!!! warning
-After an upgrade, some python requirements may have changed. Follow the
-instructions in the Python [requirements](#python-requirements-installation)
-section to update your python packages.
-
-!!! note
-    You can find some additional information about how to use Ansible's collections on the following Ansible pages:
+After an upgrade, some python requirements may have changed. Make sure to also update the Python requirements for the same version:
 
-    - [Ansible collection user guide](https://docs.ansible.com/ansible/latest/user_guide/collections_using.html)
-    - [Ansible User guide](https://docs.ansible.com/ansible/latest/user_guide/index.html)
+```shell
+pip install pyavd[ansible]
```

Comment:
May need to hardcode the version here?

Reply:
There is no version set on the galaxy upgrade above, so "latest".

Reply:
Could a rare corner case like this occur:
you are using a python version where the latest version of published pyavd to pypi is lower than the collection version?

Maybe the text above `Make sure to also update the Python requirements for the same version:` is enough

Reply:
I have added a static version to be sure.